src/App/components/TextField/index.vue

```diff
@@ -0,0 +1,15 @@
+<template lang="jade">
+  <input type="text" v-bind:value="value"  v-on:input="$emit('input', $event.target.value)" :placeholder="placeholder" />
```

Comment:
This is a [special syntax](https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components) to allow for the `v-model` binding to work on the `TextField` component.